scripts/quests/bastok/Return_to_the_Depths.lua

```diff
@@ -0,0 +1,256 @@
+-----------------------------------
+-- Return to the Depths
+-- Ayame
+-- Log ID = 1
+-- Quest ID = 78
+-----------------------------------
+require('scripts/globals/items')
+require('scripts/globals/quests')
+require('scripts/globals/interaction/quest')
+require('scripts/globals/npc_util')
+require("scripts/globals/status")
+require('scripts/globals/titles')
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.BASTOK, xi.quest.id.bastok.RETURN_TO_THE_DEPTHS)
+
+quest.reward =
+{
+    fame     = 10,
+    fameArea = xi.quest.fame_area.BASTOK,
+    item     = xi.items.BOWYER_RING,
+    title    = xi.title.GOBLIN_IN_DISGUISE,
+}
+
+quest.sections =
+{
+    -- Available
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and
+            player:getFameLevel(xi.quest.fame_area.BASTOK) >= 5 and
+            player:hasCompletedQuest(xi.quest.log_id.BASTOK, xi.quest.id.bastok.A_QUESTION_OF_FAITH)
+        end,
+
+        [xi.zone.METALWORKS] =
+        {
+            ['Ayame'] = quest:progressEvent(879),
+
+            onEventFinish = 
+            {
+                [879] = function(player, csid, option, npc)
+                    -- if option == 1 then
+                        quest:begin(player)
+                        quest:setVar(player, 'prog', 0)
```

Comment:
prog should be set to 0 by default. no need for this line! :)
Should also uncomment the option 1, as the player has the option to accept or deny this quest during the dialogue with Ayame.

Reply:
I can update the prog.  However, there's no option for this one.  When you talk to Ayame, you just get a 4-line c/s with no options.